### PR TITLE
[REVIEW] Add in JNI support for the repeat command [skip-ci]

### PR DESCRIPTION
**Pull request by revans2**
Adds in java support for the repeat command.

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

File: java/src/main/java/ai/rapids/cudf/Table.java [hunk 1]
```diff
@@ -859,6 +865,41 @@ public ColumnVector interleaveColumns() {
...
+   * Repeat the contents of this table count times.
+   * @param count the number of times to repeat each row.
+   * @return the new Table.
+   */
+  public Table repeat(int count) {
+    return new Table(repeatStaticCount(this.nativeHandle, count));
+  }
+
+  /**
+   * Create a new table by repeating the rows of this table. The number if
```
Comment (jlowe) [inline java/src/main/java/ai/rapids/cudf/Table.java:878]:
```suggestion
   * Create a new table by repeating the rows of this table. The number of
```

File: java/src/main/java/ai/rapids/cudf/Table.java [hunk 2]
```diff
@@ -859,6 +865,41 @@ public ColumnVector interleaveColumns() {
...
+   *               Integer type, and must have one entry for each row in the table.
+   * @return the new Table.
+   * @throws CudfException on any error.
+   */
+  public Table repeat(ColumnVector counts) {
+    return repeat(counts, true);
+  }
+
+  /**
+   * Create a new table by repeating the rows of this table. The number if
```
Comment (jlowe) [inline java/src/main/java/ai/rapids/cudf/Table.java:890]:
```suggestion
   * Create a new table by repeating the rows of this table. The number of
```

File: java/src/main/java/ai/rapids/cudf/Table.java [hunk 3]
```diff
@@ -859,6 +865,41 @@ public ColumnVector interleaveColumns() {
...
+   * repetitions of each element is defined by the corresponding value in counts.
+   * @param counts the number of times to repeat each row. Cannot have nulls, must be an
+   *               Integer type, and must have one entry for each row in the table.
+   * @param checkCount should counts be checked for errors before processing. be careful if you
```
Comment (jlowe) [inline java/src/main/java/ai/rapids/cudf/Table.java:894]:
```suggestion
   * @param checkCount should counts be checked for errors before processing. Be careful if you
```

File: java/src/main/java/ai/rapids/cudf/Table.java [hunk 4]
```diff
@@ -859,6 +865,41 @@ public ColumnVector interleaveColumns() {
     return new ColumnVector(interleaveColumns(this.nativeHandle));
   }
 
+  /**
+   * Repeat the contents of this table count times.
```
Comment (jlowe) [inline java/src/main/java/ai/rapids/cudf/Table.java:869]: It's ambiguous whether the repetition is by row or by the entire table.  For example, a repeat count of 2 on 1, 2, 3 could be either 1, 1, 2, 2, 3, 3 or 1, 2, 3, 1, 2, 3 depending on whether it's interpreted to be a by row or by table repetition.  The docs says it repeats the table which implies the latter, but the unit test shows it's actually the former.

**Comment by revans2** [comment]
@jlowe please take another look.

**Pull request merged**